lib/active_job/queue_adapters/resque_ext.rb

```diff
@@ -77,6 +77,10 @@ def discard_job(job, jobs_relation)
     resque_jobs_for(jobs_relation).discard(job)
   end
 
+  def dispatch_job(job, jobs_relation)
+    # noop
+  end
```

Comment:
I think we don't want this here because we already have a way to express that a method is not supported by an adapter and does nothing, via the `Adapter` module. In the `Adapter` module, you get methods such as 
```ruby
  def workers
    if exposes_workers?
      raise_incompatible_adapter_error_from :workers
    end
  end
```
that, in the case of Resque right now, it just returns `nil` because `exposes_workers?` is `false` for resque. We want something similar in this case. If the adapter supports the `blocked` status, then it should provide this method. If it doesn't support it, we don't need to add it and no-op.